Provider/src/main/java/dev/openfeature/contrib/providers/ConfidenceFeatureProvider.kt

```diff
@@ -184,15 +217,25 @@ class ConfidenceFeatureProvider private constructor(
     companion object {
         private class ConfidenceMetadata(override var name: String? = "confidence") : ProviderMetadata
 
+        fun isStorageEmpty(
+            context: Context
+        ): Boolean {
+            val storage = StorageFileCache.create(context)
+            val data = storage.read()
+            return data == null
+        }
+
         @Suppress("LongParameterList")
         fun create(
             context: Context,
             clientSecret: String,
             region: ConfidenceRegion = ConfidenceRegion.EUROPE,
+            initialisationStrategy: InitialisationStrategy = InitialisationStrategy.FetchAndActivate,
             hooks: List<Hook<*>> = listOf(),
             client: ConfidenceClient? = null,
             metadata: ProviderMetadata = ConfidenceMetadata(),
```

Comment:
For another time: We should probably align on when we use default params. In this `create` function or in the constructor.